CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+# Contributing
+
+First of all, thank you for contributing to the Software Gardening Almanac! 🎉 💯 🪴
+We're absolutely grateful for the kindness of your effort applied to grow this project into the best that it can be.
+
+This document contains guidelines on how to most effectively contribute to this project.
+
+If you are stuck or would like clarifications, please feel free to ask any questions or for help.
+
+## Code of conduct
+
+This project is governed by our [Code of Conduct (CoC) policy](https://github.com/software-gardening/almanac?tab=coc-ov-file) (located [here](https://github.com/software-gardening/.github/blob/main/CODE_OF_CONDUCT.md)).
+By participating, you are expected to uphold this code.
+Please report unacceptable behavior by following the procedures listed under the [CoC Enforcement section](https://github.com/software-gardening/almanac?tab=coc-ov-file#enforcement).
+
+## Security
+
+Please see our [Security policy](https://github.com/software-gardening/almanac?tab=security-ov-file) (located [here](https://github.com/software-gardening/.github/blob/main/SECURITY.md)) for more information on security practices and recommendations associated with this project.
+
+## Quick links
+
+- Documentation: <https://github.com/software-gardening/almanac>
+- Issue tracker: <https://github.com/software-gardening/almanac/issues>
+
+## Process
+
+### Reporting bugs or suggesting enhancements
+
+We’re deeply committed to a smooth and intuitive user experience which helps people benefit from the content found within this project.
+This commitment requires a good relationship and open communication with our users.
+
+We encourage you to report bugs or propose enhancements to improve the Software Gardening Almanac as a [GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) associated with the repository.
+
+First, figure out if your proposal is already implemented by reading existing issues or pull requests!
+Next, check the issues (<https://github.com/software-gardening/almanac/issues>) to see if someone else has already reported the bug or proposed the enhancement you have in mind.
+If you do find the suggestion, please comment on the existing issue noting that you are also interested in the functionality.
+If you do not find the suggestion, please open a new issue and document the specific enhancement and why it would be helpful for your particular use case.
+
+Specifically, the following may help when it comes to addressing the issue:
+
+- The version of the Software Gardening Almanac you're referencing.
```

Comment:
It might be helpful to add this as an ISSUE_TEMPLATE as well, since many will not thoroughly read the CONTRIBUTING guide

Reply:
Great idea, thanks, I'll add these into an ISSUE_TEMPLATE with another addition!